velox/connectors/CMakeLists.txt

```diff
@@ -15,4 +15,6 @@ add_library(velox_connector Connector.cpp)
 
 target_link_libraries(velox_connector velox_config velox_vector)
 
-add_subdirectory(hive)
+if(${VELOX_ENABLE_HIVE_CONNECTOR} OR ${VELOX_BUILD_TESTING})
```

Comment:
I like Predro's suggestion above to use a single variable here and make all the decisions in the top-level CMake file.